crates/sui-json-rpc/src/transaction_execution_api.rs

```diff
@@ -87,7 +87,7 @@ impl TransactionExecutionApi {
         let txn = Transaction::from_generic_sig_data(tx_data, Intent::sui_transaction(), sigs);
         let digest = *txn.digest();
         let raw_transaction = if opts.show_raw_input {
-            bcs::to_bytes(txn.data())?
+            tx_bytes.to_vec()?
```

Comment:
I think this will break the client side parsing. cc @patrickkuo

Reply:
this is ok, they are both bytes of `TransactionData`